### PR TITLE
docs: add disclaimer about creating >30d invites

### PR DESCRIPTION
**Pull request by onerandomusername**
## Summary

Adds simple documentation note about the new 30d invite link day limit.

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `pdm lint`
    - [ ] I have type-checked the code by running `pdm pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: disnake/abc.py [hunk 1]
```diff
@@ -1290,9 +1290,16 @@ async def create_invite(
         max_age: :class:`int`
             How long the invite should last in seconds. If it's 0 then the invite
             doesn't expire. Defaults to ``0``.
+
+            .. warning::
+
+                If the guild is not a Community guild (has ``COMMUNITY`` in :attr:`.Guild.features`),
+                this must be set to a time between ``0`` and ``2_592_000`` seconds.
```
Comment (Victorsitou) [inline disnake/abc.py:1297]: imo it looks better without the underscores

File: disnake/abc.py [hunk 2]
```diff
@@ -1290,9 +1290,16 @@ async def create_invite(
...
+
         max_uses: :class:`int`
             How many uses the invite could be used for. If it's 0 then there
             are unlimited uses. Defaults to ``0``.
+
```
Comment (Victorsitou) [inline disnake/abc.py:1302]: this is not needed

Comment (onerandomusername) [reply disnake/abc.py:1302]: removing this turns the html into this 
image

Comment (Victorsitou) [reply disnake/abc.py:1302]: what..... fine

File: disnake/abc.py (hunk 1 above)
Comment (onerandomusername) [reply disnake/abc.py]: switched to commas in https://github.com/DisnakeDev/disnake/pull/1056/commits/9fef8f2cb74c76c4c1bff54a0558c750bacc364f

File: disnake/abc.py (hunk 2 above)
Comment (onerandomusername) [reply disnake/abc.py:1301]: oh! I didn't look closely on which line this was commented on!

Comment (onerandomusername) [reply disnake/abc.py]: https://github.com/DisnakeDev/disnake/pull/1056/commits/e16ea2a95206e1193563f50cc91fb9feea783da3

File: disnake/abc.py (hunk 1 above)
Comment (onerandomusername) [reply disnake/abc.py]: removed commas entirely as per discord discussion 
[`8453d33` (#1056)](https://github.com/DisnakeDev/disnake/pull/1056/commits/8453d33e71808739ac50db537f0332f177d12829)

**Review by Victorsitou — APPROVED** [review]

File: disnake/abc.py [hunk 3]
```diff
@@ -1290,6 +1290,12 @@ async def create_invite(
         max_age: :class:`int`
             How long the invite should last in seconds. If it's 0 then the invite
             doesn't expire. Defaults to ``0``.
+
+            .. warning::
+
+                If the guild is not a Community guild (has ``COMMUNITY`` in :attr:`.Guild.features`),
+                this must be set to a time between ``0`` and ``2592000`` seconds.
```
Comment (shiftinv) [inline disnake/abc.py:1297]:
```suggestion
                this must be set to a time between ``1`` and ``2592000`` seconds.
```

Comment (Victorsitou) [reply disnake/abc.py:1297]: I was also thinking of something like: "this must not be set to ``0``."

**Review by shiftinv — APPROVED** [review]

**Pull request merged**